modules/simple_forms_api/app/models/simple_forms_api/vba_21_0966.rb

```diff
@@ -67,7 +67,12 @@ def submission_date_config
     def track_user_identity(confirmation_number)
       identity = data['preparer_identification']
       StatsD.increment("#{STATS_KEY}.#{identity}")
+      benefit_types = data['benefit_selection'].map do |benefit_type, is_selected|
+        benefit_type if is_selected
+      end.compact.join(', ')
+
       Rails.logger.info('Simple forms api - 21-0966 submission user identity', identity:, confirmation_number:)
+      Rails.logger.info('Simple forms api - 21-0966 submission benefit types', benefit_types:, confirmation_number:)
```

Comment:
I'm pretty on the fence about it so happy to go with your suggestion if you think it's preferable. I don't know if noisy logs make a huge difference since I don't know if anyone is tailing the logs and might be annoyed by it. I feel like this does make it harder to search against because if we, for example, want to search for `"submission user identity"` across all forms, we'd have to remember that 21-0966 is a special case and would have to change our search logic. How strongly do you feel that we should combine these logs?

Reply:
The primary issue with noisy logs is the challenge in quickly pinpointing critical events or errors. If you can reduce logs, I'm all for it

Reply:
Ok I've combined the logs in a slightly different way but now it will just produce one log entry per submission. Thanks @LindseySaari and @pennja !